README.md

```diff
@@ -0,0 +1,222 @@
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <h3><b>All In Campsters</b></h3>
+  <h4>Module 1: Capstone Project</h4>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+  - [🎥 Video Description](#video-description)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Capstone Project 1] <a name="about-project"></a>
+
+All In Campsters is my opportunity to put what i've learned during module 1 in practice.
+
+## 🛠 Built With <a name="built-with"></a>
+HTML, CSS, NodeJS
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Structure</summary>
+  <ul>
+    <li><a href="https://html.com/">HTML</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Style</summary>
+  <ul>
+    <li><a href="https://www.w3schools.com/css/">CSS</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Linters</summary>
+  <ul>
+    <li><a href="https://webhint.io/">Webhint</a></li>
+    <li><a href="https://stylelint.io/">Stylelint</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+-Linter is a tool to help you improve your code, it analizes the source code looking for problems.
+
+-Mobile first design: it's addapted to mobile screens.
+
+-Dynamic HTML: Write HTML Code using JavaScript.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+-Link: (https://codedit334.github.io/Module1_Capstone-Project/index.html)[https://codedit334.github.io/Module1_Capstone-Project/index.html].
+
```

Comment:
- [ ] [REQUIRED] Good job so far! Kindly your deployment link is not wprking, please change it to correct one 🙏🏼 

![image](https://user-images.githubusercontent.com/63915024/217051201-d63ec3bf-c9f1-479c-be66-c0a29f5246a7.png)